src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -582,7 +582,7 @@ rpm_ostree_compose_context_new (const char    *treefile_pathstr,
   /* Test whether or not bwrap is going to work - we will fail inside e.g. a Docker
    * container without --privileged or userns exposed.
    */
-  if (!opt_print_only && !rpmostree_bwrap_selftest (error))
+  if (!(opt_download_only || opt_download_only_rpms) && !rpmostree_bwrap_selftest (error))
```

Comment:
But don't we still need the check for `opt_print_only`?

Reply:
Ahh whoops, I didn't separate that commit from the next correctly.